src/reset.css

```diff
@@ -0,0 +1,15 @@
+/* #region reset */
+html {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
why override the font here? it can be specified immediately in html

Reply:
not fixed